src/RangeEnclosures.jl

```diff
@@ -1,6 +1,7 @@
 module RangeEnclosures
 
 using DynamicPolynomials, Requires, Reexport
+@reexport using ForwardDiff
```

Comment:
do you have some specific function in mind to be used when loading `RangeEnclosures.jl`? otherwise i would not reexport it, but i don't have any strong opinion against it

Reply:
You need to re-export it to have the default value of `df`.

The derivative is (will be) also needed in the mean form evaluatoon method.

Alternatively, one could remove it and document that the user must provide a derivative function (similar to `backend` for  SOS).

since (if?) ForwardDiff is pretty lightweight,  maybe we can keep it? Battery included is nice as long as it's not too heavy

Reply:
sure